cassandane/Cassandane/Cyrus/Quota.pm

```diff
@@ -1134,7 +1134,6 @@ sub test_quota_f_no_improved_mboxlist_sort
         'user2@example.com',
     );
 
-    $self->{instance}->{config}->set('improved_mboxlist_sort', 'no');
```

Comment:
Same for `sub test_quota_f_no_improved_mboxlist_sort`, no special behaviour for it to test anymore, so we don't need the test.